### ISSUE TITLE
18.58 - "Can't load content - Sign in" immediately after content is loaded

### ISSUE THREAD

**Issue by hiroo916** [yuliskov/SmartTubeNext#2746]
Previous version was working. I did have some problems with the subtitles so I updated since the new version had some fixes for that.

Right after update, it will load the Home screen content, but after a flash glimpse of those, it immediately goes to the "Can't load content 1) check date time 2) network 3) proxy. etc - Sign in"

I tried revalidating my log-in via web browser on computer but it still does the same thing.  Tried rebooting device, same thing.

Nvidia Shield 2017.

**Comment by dnyplaya** [comment]
Same bug here on Fire Cube (2nd gen).

**Comment by elijahmcneish** [comment]
Same on Chromecast with Google TV. It only seems to be occurring in the Home section. All the others are fine.

**Comment by hiroo916** [comment]
Hmm, I've tried it a few times over the past few hours and it kept having the same problem, but I just went to try it again and loading Home works now.  Don't know what changed.

**Comment by Navneet-Kumar-Singh** [comment]
> Hmm, I've tried it a few times over the past few hours and it kept having the same problem, but I just went to try it again and loading Home works now. Don't know what changed.

Home worked again but now on checking again the issue returned

**Comment by emtee40** [comment]
Same here. New beta will drop linked account. Reverted to Stable and no issues.

Amazon FireStick 4k Max - AFTKA - kara

**Comment by NyctophobicLemur** [comment]
I have the same issue with 18.58.  If I navigate fast enough to the video, it will not ask.

Nvidia Shield TV Model: P2897

**Comment by hiroo916** [comment]
> Hmm, I've tried it a few times over the past few hours and it kept having the same problem, but I just went to try it again and loading Home works now. Don't know what changed.

So it worked last night but then tried in the morning and Home now has the same problem again.

**Comment by joefast77** [comment]
I have the same issue on my nvidia shield pro .

**Comment by bibs213** [comment]
Same issue - Amazon FireStick 4k Max - AFTKA

**Comment by thebudman420** [comment]
This does this for me on the home tab.

Doesn't do this on any other tab.

Only does this on the account that shows my email. This account can watch age restricted. My other account doesn't let me. Even though i could upload age restricted videos with the account if i wanted to.

My other account is a YouTube creators account or something that says username random 4 digit number and @pages.plusgoogle.com

That account works perfectly fine. That is the account google doesn't let watch age restricted. "Not a problem with the app" Google policy or something.

They are both off my same Google login basically. But different history, playlist, likes and all that.

I mostly use the account i can watch age restricted content on. The broken account.

Searching for videos work. I can get to other tabs, history, likes, playlist and everything else.

Only the home tab is broken. On accounts that show your email in the login part of the interface.

**Comment by thebudman420** [comment]
Editing my previous comment broke.

When i use one user the where i left off bar shows on the other user name on my user that shows email in the app.

Does this the other way around too.

On one user it will show all the way watched if i switch and partially warched when i switch back. The user with email under login doesn't show the watch duration after watching part until i switch user. This shows up on that user and when i switch back.

I think this is glitching the home page.too.

There is a few other details i was going to add. Typed it out in above comment and GitHub glitched and erased half.

**Comment by thebudman420** [comment]
You fixed the home page for me. I just got offered the update.

The actual history of two users is still separate.

There is now only a bug with the watch duration bar from the channels page affecting two users unlike before.

The trick to switching tabs after watching shows the duration slider bar. However the duration slider bar is duplicated on two users.

History remains separate. Figured it out. Haven't checked the channels tab or news or music tabs or other tabs to make sure something isn't affecting them too.
